ssc/cmod_merchantplant_eqns.cpp

```diff
@@ -153,7 +153,7 @@ bool mp_ancillary_services(ssc_data_t data)
 		std::vector<ssc_number_t> ancillary_services3_revenue(nsteps, 0.0);
 		std::vector<ssc_number_t> ancillary_services4_revenue(nsteps, 0.0);
 
-		if (!ancillary_services_success)
+//		if (!ancillary_services_success)
 		{
```

Comment:
Not sure why the bracket is still here if the if statement has been commented out.

Reply:
Updated confusing indentation and fixed brackets. Thanks!